### PR TITLE
P1 text utility

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
you could delete the commented code, other than that it looks good to me. Good UI ✨

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.input{
+  border:none;
+  border-bottom: 3px solid ;
```
Comment (dev-ashishsethi) [inline css/components/input.css:3]: color is missing

**Review by dev-ashishsethi — APPROVED** [review]
good to go

File: css/components/input.css (hunk 1 above)
Comment (ritikaawasthy) [reply css/components/input.css:3]: color would be added from class

**Pull request merged**